migrations/postgres/V9__create_environment_table.sql

```diff
@@ -0,0 +1,14 @@
+---- Create environmnet table ----
+
+CREATE TABLE IF NOT EXISTS environments (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    name VARCHAR(255) NOT NULL UNIQUE,
+    type VARCHAR(50) NOT NULL,
+    slug VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
no need to be unique. IT must be unique across a tenant not across all tenants so we can put a constraint on tenant_id+slug maybe

Reply:
For now we can just remove the unique check here and keep it in the service layer

Reply:
fixed